### DISCUSSION TITLE
Module custom icon 1.11.26

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5229] [General]
Hello again, while editing a module I can see a select box named `extra_lp_icon` with no options, digging into the code I found out that the select is filled with icons placed inside the lp_icons inside the theme folder, so I tried to put some images and now I can see the options.

The problem is that even if I save the form selecting an icon, nothing happens, seems that this form field is not saved, I can't even find references to the `extra_lp_icon` inside the code other than the one that populates the select.
Also checked the `c_lp` table inside the database and can't find that field.

Can someone confirm that this options is disabled or still in development? If not, what am I doing wrong?

Thanks in advance

**Comment by Kaneda-1** [comment]
Hello @a-mian-lateral have you tried cleaning cache in the browser and in Chamilo platform?

**Reply by a-mian-lateral** [reply]
As a full stack developer that's the first thing I try when something does not work

**Comment by nosolored** [comment]
Hello @a-mian-lateral,
I just tried it in a Chamilo 1.11.26 platform (updated with latest Github code) and it works
The selected icon is shown in learnpath list

**Reply by a-mian-lateral** [reply]
Are there some limits on the resolution, size or format of the image?

**Reply by LudiscapeXApi** [reply]
Yes 104px by 104px
But you can change the defaut size here :
main/lp/learnpath.class.php line 12136
`$temp->resize(104);`
